scripts/development_fixtures.py

```diff
@@ -535,6 +535,14 @@ def _create_reader_studies(users):
         color_space="RGB",
     )
     image.save()
+    annotation_interface = ComponentInterface(
+        store_in_database=True,
+        relative_path="annotation.json",
+        slug="annotation",
+        title="Annotation",
+        kind=ComponentInterface.Kind.TWO_D_BOUNDING_BOX,
+    )
+    annotation_interface.save()
```

Comment:
This should likely be cherry-picked on top of:
- https://github.com/comic/grand-challenge.org/pull/2715